project_management/credit_requests/turing_mybinder_azure_credit_request_2022-03.md

```diff
@@ -0,0 +1,104 @@
+# Turing Binder Federation Membership - Credit Request
+
+## Project Title
+
+Turing membership of the International Binder Federation
+
+## Project PI
+
+Callum Mole; cmole@turing.ac.uk
```

Comment:
It might be useful to also list @malvikasharan here?

Reply:
@sgibson91 have done so.